sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosPatchOperations.java

```diff
@@ -291,6 +291,48 @@ public CosmosPatchOperations increment(String path, double value) {
         return this;
     }
 
+    /**
+     * The operator will first check if all the segments in the path provided (in the request) exist in the document.
+     * In case they do not, the operator will create them and set the provided value at the leaf segment of the provided path.
+     * In case the path already exists, the operator will not perform any modifications to the existing document at all.
+     *
+     * For the above JSON, we can have something like this:
+     * <code>
+     *     CosmosPatchOperations cosmosPatch = CosmosPatchOperations.create();
+     *     cosmosPatch.createIfNotExists("/a/x/y", 15); // will add the path /a/x/y and set the value to 15
+     *     cosmosPatch.createIfNotExists("/a/b/c", "new value"); // will not perform any modification as the path already exists
```

Comment:
The documentation says if all the segments in the path provided exist. however, in this case, /b/c does not exist. Is my understanding incorrect?